.github/workflows/_binary-build-linux.yml

```diff
@@ -11,6 +11,10 @@ on:
         required: true
         type: string
         description: The build environment
+      runs_on:
```

Comment:
We should not need this parameter here or 
Use following params for this:
```
required: false
type: string
default: "linux.12xlarge"
description: Hardware to run this job on.
```
This way all other linux jobs still run on same hardware.

Reply:
Negate my delete comment. Builds can be done on x86_64 CPU instances (not requiring GPU) where testing needs to changes instances for particular arch types (Difference GPUs). Aarch64 is different as the build and test need to be on on an aarch64 instance. As I would like to, eventually, use only one build workflow, will add the default option and remove the run_as in the template if it is not aarch64.